src/stock_server.c

```
@@ -1,34 +1,372 @@
 /*
- * stock_server.c - server lifecycle stubs
+ * stock_server.c - baseline server lifecycle
  */
+#include "csapp.h"
 #include "stock_server.h"
 
+#define STOCK_WORKER_COUNT 8
+#define STOCK_QUEUE_SIZE 64
+#define STOCK_QUEUE_STOP -1
```

Clang (14.0.6)

[warning] macro replacement list should be enclosed in parentheses

(bugprone-macro-parentheses)